### PR TITLE
Rework coupon descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary
This pull request reworks the coupon descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/Coupon/Coupon.yaml [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
-description: Coupons and Discounts.
+description: Coupons and discounts.
 required:
   - discount
   - issuedTime
 properties:
   id:
-    description: Coupon's ID a.k.a redemption code.
+    description: |-
+      Coupon identifier string. This value is also know as the coupon
```
Comment (ashkarpetin) [inline openapi/components/schemas/Coupon/Coupon.yaml:9]:
```suggestion
      Coupon identifier string. This value is also known as the coupon
```

File: openapi/components/schemas/Coupon/CouponRestrictions/restrict-to-subscriptions.yaml [hunk 2]
```diff
@@ -8,6 +8,6 @@ allOf:
     properties:
       subscriptionIds:
         type: array
-        description: Order IDs coupon can be applied to.
+        description: Order IDs on which a coupon can be applied.
```
Comment (Boydbueno) [inline openapi/components/schemas/Coupon/CouponRestrictions/restrict-to-subscriptions.yaml:11]:
```suggestion
        description: Subscription IDs on which a coupon can be applied.
```

**Review by Boydbueno — COMMENTED** [review]
Looking good! I spotted one mistake that has been in the api definitions for a while.

I also noticed that the word 'successfully' is quite commonly used for some descriptions. For example: `List of coupon redemptions retrieved successfully.`

Do you think the addition of 'successfully' adds anything?

**Comment by noon-dawg** [comment]
Thanks for the review @Boydbueno 🙏 

> I also noticed that the word 'successfully' is quite commonly used for some descriptions. 

Awesome input!! 'successfully' adds nothing to the these descriptions. I'll create a new PR to remove all mentions after this and the disputes PR are merged 👍

**Review by Boydbueno — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Pull request merged**